feat: create sms_send in comunication_services

luizpcardoso opened the pull request:
add the libraty request in requirements.txt
the comunication_services.py contains the functions used to communicate with the user

luizpcardoso committed:
feat: create sms_send in comunication_services

add the libraty request in requirements.txt
the comunication_services.py contains the functions used to communicate with the user

penguinuux commented on app/services/comunication_services.py:
```diff
@@ -0,0 +1,18 @@
+from os import getenv
+import requests
+import json
```
Remove unused imports.

penguinuux commented on .env.example:
```diff
@@ -1,3 +1,4 @@
 FLASK_ENV=development
 DATABASE_URI=postgresql://YOUR_USER:YOUR_PASSWORD@localhost:5432/YOUR_DATABASE
 SECRET=
+SMS_URI_KEY=
```
Add a new line in the end of file.

marcelosj3 replied:
Probably this is due because of the lack of using `black .`

luizpcardoso replied:
Done.

penguinuux commented on app/services/comunication_services.py:
```diff
@@ -0,0 +1,18 @@
+    "msg" : text
+    }
+    req = requests.post(SMS_URI, data=data)
+    
```
Add a new line in the end of file.

luizpcardoso replied:
Done.

luizpcardoso committed:
fix: fixed useless imports, return and last lines

luizpcardoso committed:
updated and merge develop in branch

Merge branch 'develop' of github.com:penguinuux/rain-forecast-risk-alert into feature/message-model-smssend

luizpcardoso committed:
fix: isort and black apply
